ProcessMaker/screen-builder
# Feature/FOUR-18114:STORY Record List from Collections UI Update)

## How to Test
- Login PM4
- Go to Designer->Screens
- Create or Update some screen
- Drag Record List control
- Select a Variable or Collection
- Add some columns
- Go to DESIGN tab and choose "Modern" in dropdown Table Style
- Click on Preview(Classic Mode should display Table as always, and Modern mode should display new Styles)


## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-18114

## Code Review Checklist
